### DISCUSSION TITLE
Allow plugins to hook into Find/Replace functionality

### DISCUSSION THREAD

**Discussion by engram-design** [craftcms/cms#8625] [Ideas]
Related to https://github.com/verbb/super-table/issues/114, can other fields hook into Find & Replace functionality? We'd like to be able to copy what Matrix does with Super Table, but all functionality related to that seems to be hardcoded https://github.com/craftcms/cms/blob/develop/src/queue/jobs/FindAndReplace.php#L58.

I imagine this would be useful for other plugins that have their own content tables.

**Comment by GaryReckard** [comment]
Would love this as well!

**Comment by nettum** [comment]
Same here!  
Have a lot of content inside super-tables and it would be awesome if that could hook into the S&R functionality without doing it manually ourselves.
